.github/workflows/lint.yml

```diff
@@ -62,3 +64,19 @@ jobs:
       - name: Run Version Checks
         run: |
           ./build_tools/github_actions/lint_version.sh
+
+  buildifier-check:
+    if: "github.event_name == 'pull_request'"
+    runs-on: ${{ github.repository == 'openxla/stablehlo' && 'ubuntu-22.04-64core' ||  'ubuntu-22.04'  }}
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checking out repository
+        uses: actions/checkout@v4
+
+      - name: Install buildifier
+        run: go install github.com/bazelbuild/buildtools/buildifier@433ea85 # 6.4.0
+
+      - name: Run buildifier
+        run: buildifier --mode=check -r . || (echo "Please run buildifier --mode=fix -r . in the repository's root directory" && exit 1)
```

Comment:
should we run the lint as well?

Reply:
You have to ignore the LLVM directory in that case though.

Reply:
Yep, and yep. Thanks! Ended up writing a small script because it became kinda onerous to have it in the `run:` line.